com.unity.netcode.gameobjects/Tests/Runtime/Physics/NetworkRigidbody2DTest.cs

```diff
@@ -68,15 +68,20 @@ public IEnumerator TestRigidbodyKinematicEnableDisable()
             // despawn the server player, (but keep it around on the server)
             serverPlayer.GetComponent<NetworkObject>().Despawn(false);
 
-            yield return NetworkRigidbodyTestBase.WaitForFrames(5);
+            yield return WaitForTicks(m_ServerNetworkManager, 5);
 
             // This should equal Kinematic
             Assert.IsTrue(serverPlayer.GetComponent<Rigidbody2D>().isKinematic == Kinematic);
 
-            yield return NetworkRigidbodyTestBase.WaitForFrames(5);
+            yield return WaitForTicks(m_ServerNetworkManager, 5);
 
             Assert.IsTrue(clientPlayer == null); // safety check that object is actually despawned.
         }
 
+        public IEnumerator WaitForTicks(NetworkManager networkManager, int count)
```

Comment:
Could this be moved to our standard testing code vs. just being here for RigidBody tests?

Reply:
could be but this PR is just bringing it back into `develop`.
I think @NoelStephensUnity might be interested in generalizing `WaitForTicks` later but potentially under another PR though.